tests/unit/RequestConflictUtilsTest.ts

```diff
@@ -1,11 +1,11 @@
-import resolveDuplicationConflictAction from '@libs/actions/RequestConflictUtils';
+import {filterRequestByCommand, resolveDuplicationConflictAction} from '@libs/actions/RequestConflictUtils';
 import type {WriteCommand} from '@libs/API/types';
 
 describe('RequestConflictUtils', () => {
     it.each([['OpenApp'], ['ReconnectApp']])('resolveDuplicationConflictAction when %s do not exist in the queue should push %i', (command) => {
         const persistedRequests = [{command: 'OpenReport'}, {command: 'AddComment'}, {command: 'CloseAccount'}];
         const commandToFind = command as WriteCommand;
-        const result = resolveDuplicationConflictAction(persistedRequests, commandToFind);
+        const result = resolveDuplicationConflictAction(persistedRequests, filterRequestByCommand(commandToFind));
```

Comment:
I feel this confusing, why not just use 
`(request) => request.command === WRITE_COMMANDS.UPDATE_COMMENT`

The name of the function `filterRequestByCommand` implies you are filtering something and returning a result, but it is creating a function